docs/references/kontena-yml.md

```diff
@@ -421,30 +442,7 @@ volumes_from:
 ```
 (`-%%s` will be replaced with container number; for example, the first service container will get volumes from wordpress-1, the second from wordpress-2, etc.)
 
-#### log_driver
-> **Note:** Version 1 only. In version 2 use [logging](#logging) options
-
-Specify the log driver for Docker to use with all containers of this service. For details on available drivers and their configs see [Docker log drivers](https://docs.docker.com/reference/logging/overview/)
-
-#### log_opts
-> **Note:** Version 1 only. In version 2 use [logging](#logging) options
-
-Specify options for log driver
-
-```
-nginx:
-  image: nginx:latest
-  ports:
-    - 80:80
-  log_driver: fluentd
-  log_opt:
-    fluentd-address: 192.168.99.1:24224
-    fluentd-tag: docker.{{.Name}}
-
-```
-
 #### logging
```

Comment:
Keep the description/examples from the earlier `log_driver`/`log_opts` sections here?